wikitemplate.md

```diff
@@ -130,39 +130,69 @@ _Each start should take less than 7 seconds_
-- [ ] Verify you are able to make recurring tip and they display in tips panel
-- [ ] Verify you can tip a verified publisher
-- [ ] Verify you can tip a verified YouTube creator
-- [ ] Verify tip panel shows a verified checkmark for a verified publisher/verified YouTube creator
-- [ ] Verify tip panel shows a message about unverified publisher
-- [ ] Verify BR panel shows message about an unverified publisher
-- [ ] Verify you are able to perform a contribution
-- [ ] Verify if you disable auto-contribute you are still able to tip regular sites and YouTube creators
-- [ ] Verify that disabling Rewards and enabling it again does not lose state
-- [ ] Verify that disabling auto-contribute and enabling it again does not lose state
-- [ ] Adjust min visit/time in settings. Visit some sites and YouTube channels to verify they are added to the table after the specified settings
-- [ ] Upgrade from older version
-  - [ ] Verify the wallet balance is retained and wallet backup code isn't corrupted
-  - [ ] Verify auto-contribute list is not lost after upgrade
-  - [ ] Verify tips list is not lost after upgrade
-  - [ ] Verify wallet panel transactions list is not lost after upgrade
+- [ ] Verify you are able to create a new wallet.
+- [ ] Verify you are able to restore a wallet.
+- [ ] Verify account balance shows correct BAT and USD value.
+- [ ] Verify actions taken (claiming grant, tipping, auto-contribute) display in wallet panel.
+- [ ] Verify AC monthly budget shows correct BAT and USD value.
+- [ ] Verify you are able to exclude a publisher from the auto-contribute table and popup list of sites.
+- [ ] Verify you are able to exclude a publisher by using the toggle on the Rewards Panel.
+- [ ] Verify you are able to perform an auto contribution.
+  - [ ] Verify auto contribution is reflected in the rewards widget on the NTP.
+- [ ] Verify monthly statement shows expected data.
+- [ ] Verify when you click on the BR panel while on a site, the panel displays site specific information (site favicon, domain, attention %).
+- [ ] Verify BR panel shows message about an unverified publisher.
+- [ ] Verify one time and monthly tip banners show a message about unverified publisher.
+- [ ] Verify one time tip and monthly tip banners show a verified checkmark for a verified creator.
+- [ ] Verify when you click on `Send a tip`, the custom tip banner displays if set up.
+  - [ ] Verify custom tip banner is also displayed for monthly contribution.
+- [ ] Verify you are able to make one-time tip and they display in Tips panel.
+  - [ ] Verify tip is reflected in the rewards widget on the NTP.
+- [ ] Verify you are able to make recurring tip and they display in Monthly Contributions panel.
+  - [ ] Verify you are able to adjust your recurring tip amount from the BR panel.
+  - [ ] Verify recurring tip is reflected in the rewards widget on the NTP.
+- [ ] Verify you can tip a verified website.
```

Comment:
I would also add to verify if you tip unverified publisher that it goes in pending list

Reply:
updated, please review.